README.md

```diff
@@ -38,7 +38,7 @@ pactl list sources short
 Then, create the new device using:
 ```sh
 pacmd load-module module-null-sink sink_name=mic_denoised_out rate=48000
-pacmd load-module module-ladspa-sink sink_name=mic_raw_in sink_master=mic_denoised_out label=noise_suppressor_mono plugin=/path/to/librnnoise_ladspa.so control=50
+pacmd load-module module-ladspa-sink sink_name=mic_raw_in sink_master=mic_denoised_out label=noise_suppressor_mono plugin=/path/to/librnnoise_ladspa.so control=50,
```

Comment:
`control=50,`

Would the lack of `,` in existing installations pose a problem?

Reply:
Unfortunately yes.
If the `,` is missing you get the error message:
```
Not enough control values passed, 2 expected, 1 passed.
Failed to parse, validate or set control parameters
```

Reply:
If we don't want to break exsiting setups we could introduce something like `noise_suppressor_vad_mono`.

Reply:
Ugh, going through https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/blob/master/src/modules/module-ladspa-sink.c - I don't see any workarounds. What a great usability... How is a new parameter is supposed to be added to a plugin.

I don't like generating another shared lib either. Hmmmmm